app/controllers/posts_controller.rb

```diff
@@ -11,4 +24,10 @@ def index
   def show
     @post = Post.find_by(author_id: params[:user_id], id: params[:id])
   end
+
+  private
+
+  def post_params
+    params.require(:post).permit(:title, :text).merge(author_id: current_user.id)
+  end
```

Comment:
- [ ] Great job using strong parameters! 🚀 However, I'm unable to create new posts, it looks like you have to initialize your counters for the post to be valid! 😱  Alternatively, please consider setting a default value of 0 in your database. Every time a post is created, both likes and comments counter start at 0 and you don't have to explicitly initialize them! 🤓 🔧